mypy/nodes.py

```diff
@@ -2262,6 +2296,8 @@ class SymbolTableNode:
     # for other nodes, optionally the name of the referenced object.
     cross_ref = None  # type: Optional[str]
     # Was this node created by normalіze_type_alias?
+    #
+    # TODO: Write a better comment, this is confusing
```

Comment:
This one distinguishes ``typing.List`` and ``builtins.list`` (sorry this comment is my fault)

Reply:
Thanks for the hint! I'll update the comment.